src/main/java/org/reaktivity/rym/internal/RymInstall.java

```diff
@@ -0,0 +1,301 @@
+        contents.append("    </chain>\n");
+        contents.append("  </resolvers>\n");
+        contents.append("</ivysettings>\n");
+
+        BufferedWriter out = new BufferedWriter(new FileWriter(ivySettingsFile));
+        out.write(contents.toString());
+        out.close();
+        return ivySettingsFile;
+    }
+
+    private File createIvyFile() throws IOException
+    {
+        File ivyFile = File.createTempFile("ivy.xml", null);
+        ivyFile.deleteOnExit();
+
+        // TODO What to put for "organization" and "module" values here:
+        StringBuffer contents = new StringBuffer();
+        contents.append("<ivy-module version=\"2.0\">\n");
+        contents.append("  <info organisation=\"reaktivity\" module=\"reaktivity-deps\"/>\n");
+        contents.append("  <dependencies>\n");
+        for (Map.Entry<String, String> entry : dependencies.entrySet())
+        {
+            String[] dependencyComponents = entry.getKey().split(":");
+            contents.append(String.format("    <dependency org=\"%s\" name=\"%s\" rev=\"%s\"/>\n",
+                dependencyComponents[0],
+                dependencyComponents[1],
+                entry.getValue()));
+        }
+        contents.append("    <dependency org=\"commons-lang\" name=\"commons-lang\" rev=\"2.0\"/>\n");
+        contents.append("    <dependency org=\"commons-cli\" name=\"commons-cli\" rev=\"1.2\"/>\n");
+        contents.append("  </dependencies>\n");
+        contents.append("</ivy-module>\n");
+
+        BufferedWriter out = new BufferedWriter(new FileWriter(ivyFile));
+        out.write(contents.toString());
+        out.close();
+        return ivyFile;
+    }
+
+    private void resolveDependencies() throws ParseException, IOException
+    {
+        IvySettings ivySettings = new IvySettings();
+        org.apache.ivy.Ivy ivy = Ivy.newInstance(ivySettings);
+        ivy.configure(createIvySettingsFile());
+
+        ResolveOptions resolveOptions = new ResolveOptions();
+        resolveOptions.setLog(LOG_QUIET); // TODO Pick logging option: Always, on download only, or never.
+        resolveOptions.setOutputReport(false); // TODO Show the report? LOG_QUIET suppresses it, even if setOutputReport is true
+        ResolveReport resolveReport = ivy.resolve(createIvyFile(), resolveOptions);
+
+        if (resolveReport.hasError())
+        {
+            List<String> problems = resolveReport.getAllProblemMessages();
+            if (problems != null && !problems.isEmpty())
+            {
+                StringBuffer errorMsgs = new StringBuffer();
+                for (String problem : problems)
+                {
+                    errorMsgs.append(problem);
+                    errorMsgs.append("\n");
+                }
+                System.err.println(errorMsgs);
+            }
+        }
+        else
+        {
+            System.out.println("Dependencies were successfully resolved");
+        }
+    }
```

Comment:
Suggest returning the `ResolveReport` from this method and handle output of success / failure at caller instead.